tensorboard/compat/proto/proto_test.py

```diff
@@ -160,7 +160,29 @@
 ]
 
 
+MATCH_FAIL_MESSAGE_TEMPLATE = """
+{} and {} did not match: {}
```

Comment:
Need a newline between `match:` and `{}`, as in the original format
string, or the diff will appear wonky (the leading `---` will appear a
line early). 

But I think that it would be easier to read the error if we just let
`difflib` handle the filenames, by passing `fromfile=tf_path` and
`tofile=tb_path` kwargs to [`difflib.unified_diff`][ud]. Then the diff
headers will properly encode the paths, like this:

```diff
--- tensorflow.core.framework.allocation_description_pb2
+++ tensorboard.compat.proto.allocation_description_pb2
@@ -43,7 +43,7 @@
   java_package: "org.tensorflow.framework"
   java_outer_classname: "AllocationDescriptionProtos"
   java_multiple_files: true
-  go_package: "github.com/tensorflow/tensorflow/tensorflow/go/core/framework"
+  go_package: "github.com/tensorflow/tensorflow/tensorflow/go/core/framework/allocation_description_go_proto"
   cc_enable_arenas: true
 }
 syntax: "proto3"
```

[ud]: https://docs.python.org/3/library/difflib.html#difflib.unified_diff

Reply:
Done, I also updated the variable names for clarity, replacing actual / expected with tf / tb